README.md

```diff
@@ -1,2 +1,50 @@
-# Hello-Microverse
-My first project as a micronaut for testing my knowledge on the tools use in the program.
+Microverse_Hello
+Work on my first project in microverse as a micronaut
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello_Microverse
+
+> A repository for practicing normal github flow
+
+## Built With
+
+- HTML
+- CSS
+- Linters
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@TaiwoEnoch](https://github.com/TaiwoEnoch)
+- Twitter: [@TaiwoEnoch](https://twitter.com/TaiwoEnoch4)
+- LinkedIn: [Taiwo Enoch](https://www.linkedin.com/in/taiwo-enoch-b88550222/)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
-Good job including a readme file that helps to make your repo more professional, but please use the new [microverse readme](https://github.com/microverseinc/readme-template), and ensure that it is personalized to fit what your project is all about. You are to change the microverse logo also to a personalized one if you have any 👍

Reply:
Oh, thank you so much, I will work on it now, I am grateful.